### PR TITLE
Remove StatusHistoryFetcher

### PR DESCRIPTION
**Pull request by danidoni**
It's not reused anywhere and the same can be done in less code

### CODE REVIEW

File: src/api/app/controllers/webui/monitor_controller.rb [hunk 1]
```diff
@@ -74,8 +74,13 @@ def events
 
   private
 
+  UPPER_RANGE_LIMIT = 24 * 365
+
   def status_history(key, range)
-    MonitorControllerService::StatusHistoryFetcher.new(key, range.to_i).call
+    user_range = [UPPER_RANGE_LIMIT, range.to_i].min
+    Rails.cache.fetch("#{key}-#{user_range}", expires_in: (user_range.to_i * 3600) / 150) do
```
Comment (hennevogel) [inline src/api/app/controllers/webui/monitor_controller.rb:81]: > (user_range.to_i * 3600) / 150) 

Can you remove the magic numbers?

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/app/controllers/webui/monitor_controller.rb (hunk 1 above)
Comment (danidoni) [reply src/api/app/controllers/webui/monitor_controller.rb]: I don't know what the `150` means here...

File: src/api/app/controllers/webui/monitor_controller.rb [hunk 2]
```diff
@@ -74,8 +74,13 @@ def events
 
   private
 
+  UPPER_RANGE_LIMIT = 24 * 365
```
Comment (hennevogel) [inline src/api/app/controllers/webui/monitor_controller.rb:77]:
```suggestion
  HOURS_IN_ONE_YEAR = 8760
```

It's a constant, it's always the same...

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/app/controllers/webui/monitor_controller.rb (hunk 1 above)
Comment (hennevogel) [reply src/api/app/controllers/webui/monitor_controller.rb]: okay...

File: src/api/app/controllers/webui/monitor_controller.rb (hunk 2 above)
Comment (danidoni) [reply src/api/app/controllers/webui/monitor_controller.rb:77]: True

**Review by saraycp — APPROVED** [review]

**Review by hennevogel — APPROVED** [review]

**Pull request merged**